org/lateralgm/subframes/FontFrame.java

```diff
@@ -183,6 +186,16 @@ public void focusLost(FocusEvent e)
 		rangeList.setLayoutOrientation(JList.VERTICAL);
 		rangeList.setVisibleRowCount(6);
 
+		rangeList.getInputMap().put(KeyStroke.getKeyStroke(VK_DELETE, 0), "delete selected");
+		rangeList.getActionMap().put("delete selected", new AbstractAction()
```

Comment:
I would have actually used `"REM_RANGE"` as the key for the action since that's the key used for the text, but that's ok.